packages/web-forms/src/assets/css/style.scss

```diff
@@ -12,15 +12,15 @@
 :root {
 	--odk-font-family: Roboto, sans-serif;
 
-	--odk-base-font-size: #{map-get(pf.$font-size-props, 'text-base')};
-	--odk-heading-font-size: #{map-get(pf.$font-size-props, 'text-4xl')};
-	--odk-title-font-size: #{map-get(pf.$font-size-props, 'text-3xl')};
-	--odk-top-group-font-size: #{map-get(pf.$font-size-props, 'text-2xl')};
-	--odk-sub-group-font-size: #{map-get(pf.$font-size-props, 'text-xl')};
-	--odk-dialog-title-font-size: #{map-get(pf.$font-size-props, 'text-xl')};
-	--odk-question-font-size: #{map-get(pf.$font-size-props, 'text-lg')};
-	--odk-answer-font-size: #{map-get(pf.$font-size-props, 'text-base')};
-	--odk-hint-font-size: #{map-get(pf.$font-size-props, 'text-sm')};
+	--odk-base-font-size: #{map.get(pf.$font-size-props, 'text-base')};
```

Comment:
This fails when running `yarn build`. Can you please add `@use 'sass:map';` in L7 of this file? That should fix it.
```
✗ Build failed in 1.11s
@getodk/web-forms:build: [build:demo ] error during build:
@getodk/web-forms:build: [build:demo ] [vite:css] [sass] There is no module with the namespace "map".
@getodk/web-forms:build: [build:demo ]    ╷
@getodk/web-forms:build: [build:demo ] 15 │     --odk-base-font-size: #{map.get(pf.$font-size-props, 'text-base')};
@getodk/web-forms:build: [build:demo ]    │                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
(Seems like the CI isn't running for fork-PRs)